src/Compilers/CSharp/Portable/Symbols/PublicNullableAnnotationExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal static class PublicNullableAnnotationExtensions
+    {
+        public static NullableAnnotation ToInternalAnnotation(this CodeAnalysis.NullableAnnotation annotation)
+        {
+            Debug.Assert((CodeAnalysis.NullableAnnotation)(NullableAnnotation.Unknown + 1) == CodeAnalysis.NullableAnnotation.Unknown);
+            return annotation == CodeAnalysis.NullableAnnotation.Default ? NullableAnnotation.Unknown : (NullableAnnotation)annotation - 1;
```

Comment:
>(NullableAnnotation)annotation - 1 [](start = 104, length = 34)

Same here, especially since we're going to re-order the annotation values soon

Reply:
The ToInternal version I don't mind swapping.

---
In reply to: [265766586](https://github.com/dotnet/roslyn/pull/34017#discussion_r265766586) [](ancestors = 265766586)